tests/foreman/cli/test_variables.py

```diff
@@ -157,15 +204,17 @@ def test_negative_create(self):
 
         @steps:
 
-        1. Create a smart Variable with Invalid name and valid default value.
+        1. Create a smart Variable with Invalid name.
 
         @assert: The smart Variable is not created.
-
-        @caseautomation: notautomated
         """
+        for name in invalid_values_list():
+            with self.subTest(name):
+                with self.assertRaises(CLIReturnCodeError):
```

Comment:
nitpick: as we now support python 2.7+, you may use following syntax:
```python
with A() as X, B() as Y, C() as Z:
    do_something()
```
to avoid nested context managers.